dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageResourceHelper.java

```diff
@@ -95,9 +95,7 @@ public void saveContent(final String pageId, final List<PageContainerForm.Contai
             }
         }
 
-        if (!multiTrees.isEmpty()) {
-            multiTreeAPI.saveMultiTrees(pageId, multiTrees);
-        }
+        multiTreeAPI.saveMultiTrees(pageId, multiTrees);
     }
 
     public void saveMultiTree(final String containerId,
```

Comment:
WrapInTransaction?

Reply:
It not neccessary because this method end in a call to this one:

https://github.com/dotCMS/core/blob/master/dotCMS/src/main/java/com/dotmarketing/factories/MultiTreeFactory.java#L253